users/user_files.sls

```diff
@@ -9,6 +9,7 @@ include:
 {%- set user_files = salt['pillar.get'](('users:' ~ username ~ ':user_files'), {'enabled': False}) -%}
 {%- set user_group = salt['pillar.get'](('users:' ~ username ~ ':prime_group:name'), username) -%}
 {%- set user_home = salt['pillar.get'](('users:' ~ username ~ ':home'), current.get('home', '/home/' ~ username )) -%}
+{%- set template = salt['pillar.get'](('users:' ~ username ~ ':user_files:template'), None) -%}
```

Comment:
@jerryjvl can you rename this variable to `user_files_template` for consistency.

Reply:
Done!